code/datums/components/forensics.dm

```diff
@@ -84,9 +84,9 @@
 	if(ishuman(M))
 		var/mob/living/carbon/human/H = M
 		add_fibers(H)
-		if(H.gloves) //Check if the gloves (if any) hide fingerprints
-			var/obj/item/clothing/gloves/G = H.gloves
-			if(G.transfer_prints)
+		var/obj/item/gloves = H.get_item_by_slot(ITEM_SLOT_GLOVES)
```

Comment:
```suggestion
		var/obj/item/gloves = H.gloves
```
We're already casting to human, we can skip the getter

Reply:
wouldn't that give `item/clothing/gloves`? Because the carbon define for gloves is `var/obj/item/clothing/gloves = null`, doesn't that mean the cuffs wouldn't be put there, since they're not clothes?

To my understanding that was part of the reason the original runtimes even happened.

Reply:
The gloves var on humans is `var/obj/item/clothing/gloves`, wouldn't that exclude the cuffs since those aren't clothes?

Wouldn't that cause the runtimes we're trying to fix?

Reply:
Types in BYOND are never checked by anything at runtime. You could store a file of your dog in gloves for all the game cares.

Reply:
fair enough, I'll swap it out.